README.md

```diff
@@ -45,8 +44,7 @@ tomorrowAsString(new Date('2015-01-01')); // '2015-01-02';
 
 ## API
 
-### parse
-`String -> Date`
+### parse :: String -> Date
```

Comment:
My main thinking behind this was to make the docs for BF and date-fp to be consistent.